.github/workflows/test-tgc.yml

```diff
@@ -42,21 +40,22 @@ jobs:
       with:
         repository: ${{ github.event.inputs.owner }}/${{ github.event.inputs.repo }}
         ref: ${{ github.event.inputs.branch }}
-        path: tgc
+        path: repo
         fetch-depth: 2
     - name: Cache Go modules and build cache
       uses: actions/cache@v3
       with:
         path: |
           ~/go/pkg/mod
           ~/.cache/go-build
-        key: ${{ runner.os }}-test-${{ github.event.inputs.repo }}-${{ hashFiles('**/go.sum') }}
+        key: ${{ runner.os }}-test-${{ github.event.inputs.repo }}-${{ hashFiles('repo/go.sum', 'repo/google-beta/transport/**', 'repo/google-beta/tpgresource/**', 'repo/google-beta/acctest/**', 'repo/google-beta/envvar/**', 'repo/google-beta/sweeper/**', 'repo/google-beta/verify/**') }}
```

Comment:
Can we put this list in yaml format so it's easier to maintain?

Reply:
I ended up removing it from tgc. Bad commit on my part. Those aren't actually present on tgc. Does your ask still apply to the other?

Could you expand upon what you mean? I understand it's not very readable. Is that the concern?

Reply:
Yes, readability is my main concern. I was thinking we could have this list somewhere as

```
- repo/go.sum
- repo/google-beta/transport/**
- repo/google-beta/tpgresource/**
- repo/google-beta/acctest/**
- repo/google-beta/envvar/**
- repo/google-beta/sweeper/**
- repo/google-beta/verify/**
```

And then reference it here, but I'm not sure what the syntax to reference it would look like in an action.

Reply:
I don't have a way to improve readability after testing.. there was no way to transition this to yaml and reference it within the function (that I could find). Only way would be to store it in a string on file in our repo then read that string at runtime. I would like to avoid that scenario.. If you have implementation suggestions let me know.